### ISSUE TITLE
Add Logistic Curve chart to identify inflection point

### ISSUE THREAD

**Issue by Jeff-Lewis** [CSSEGISandData/COVID-19#637]
Can we add a logistic curve chart to show inflection point? This will tell people if the virus is still spreading exponentially or not..

Refs: https://youtu.be/Kas0tIxDvrg
https://feld.com/archives/2020/03/exponential-growth-and-covid-19.html

**Comment by bellumfatum3** [comment]
Yes please, this would be most helpful. I'm currently extracting and generating the inflection point manually.
